src/odig.ml

```diff
@@ -0,0 +1,58 @@
+open Import
+
+type t =
+  { make_cmd : string list -> Cmd.t
+  ; cache_dir : Path.t
+  }
+
+(** TODO: propose to install odig. See
+    https://github.com/ocamllabs/vscode-ocaml-platform/pull/771#discussion_r765297112 *)
+let of_sandbox (sandbox : Sandbox.t) =
+  let cmd = Sandbox.get_command sandbox "which" [ "odig" ] in
```

Comment:
We have `Cmd.which` exactly so that we don't rely on this binary existing on the user's system.